packages/suite/src/actions/bluetooth/filterOutNonResponsiveDevices.test.ts

```diff
@@ -59,20 +59,45 @@ describe(filterOutNonResponsiveDevices.name, () => {
     });
 
     it('keeps responsive devices', () => {
-        const devices: BluetoothDeviceCommon[] = [
-            createBluetoothDevice({
+        const devices: DesktopBluetoothDevice[] = [
+            createMockedBluetoothDevice({
                 id: asBluetoothDeviceId('1'),
                 name: 'DeviceA',
-                lastUpdatedTimestamp: 3_500,
+                lastUpdatedTimestamp: 7_500,
             }),
-            createBluetoothDevice({
+            createMockedBluetoothDevice({
                 id: asBluetoothDeviceId('2'),
                 name: 'DeviceA',
-                lastUpdatedTimestamp: 4_000,
+                lastUpdatedTimestamp: 7_000,
             }),
         ];
 
         const result = filterOutNonResponsiveDevices(devices);
         expect(result).toHaveLength(2);
     });
+
+    it('progressively increases the unresponsive device timeout', () => {
+        const weakSignalDevices: DesktopBluetoothDevice[] = [
+            createMockedBluetoothDevice({
+                id: asBluetoothDeviceId('1'),
+                name: 'DeviceA',
+                lastUpdatedTimestamp: 6_000,
+                rssi: -90,
+            }),
+            createMockedBluetoothDevice({
+                id: asBluetoothDeviceId('2'),
+                name: 'DeviceB',
+                lastUpdatedTimestamp: 5_000,
+                rssi: -100,
```

Comment:
To really drive this test to the main point I'd include two cases at the very limit.  
```suggestion
                lastUpdatedTimestamp: 3_000,
                rssi: -100,
            }),
            createMockedBluetoothDevice({
                id: asBluetoothDeviceId('2'),
                name: 'DeviceB',
                lastUpdatedTimestamp: 2_999,
                rssi: -100,
```

Reply:
NVM I'll make a suggestion PR shortly where I'll include this because I want to do something further...